data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,2403 @@
+		conversation
+			`You go see He Yi with the non lethal rifles to see if he has anything else to ask you. He seems very happy you managed to find them, and says, "Great job! It will be decisive in the coming war."`
+			choice
+				`	"How can a few better trained troops make a difference?"`
+				`	"Glad I could make a difference then."`
+					goto getrd
+			`	"Well... To be completely honest with you these rifles are not only intended for training purposes but I cannot tell you any more for now."`
+			
+			label getrd
+			`	"We will soon want your help with the upcoming invasion, get ready for it! Having a strong flagship is always an important asset but it is not all."`
+			choice
+				`	"The Basilisk will not be enough I take it?"`
+				`	"Okay in that case I will get ready."`
+					goto train
+			`	"The Basilisk is very strong, but it is only a prototype, it has a few weaknesses."`
+			
+			label train
+			`	"Remember, having a strong fleet and flagship does not matter much if you do not know how to use it properly! If you want to learn how to do that there are a lot of Unfettered captains looking forward to challenging you on the job board."`
+
+mission "Bring Pug Staffs"
+	invisible
+	source "Warfeed"
+	landing
+	to offer
+		has "Unfettered: Close Quarter: offered"
+	on offer
+		require "Pug Peacekeeping Staff" 10
+		dialog `Do you want to sell 10 Pug Peacekeeping Staff to the Unfettered? He Yi says they can give you <payment> for it.`
+	on accept
+		payment 2500000
+		outfit "Pug Peacekeeping Staff" -10
+		dialog `The Unfettered thank you for bringing these 10 staffs to them. They give you <payment>.`
+
+mission "Bring Coalition Weapons"
+	invisible
+	source "Warfeed"
+	landing
+	to offer
+		has "Unfettered: Close Quarter: offered"
+	on offer
+		require "Enforcer Riot Staff" 10
+		dialog `Do you want to sell 10 Enforcer Riot Staffs to the Unfettered? He Yi says they can give you <payment> for it.`
+	on accept
+		payment 2000000
+		outfit "Enforcer Riot Staff" -10
+		dialog `The Unfettered thank you for bringing these 10 staffs to them. They give you <payment>.`
+
+mission "Bring Void Rifle"
+	invisible
+	source "Warfeed"
+	to offer
+		has "Unfettered: Close Quarter: offered"
+	on offer
+		require "Void Rifle" 1
+		conversation
+			`He Yi seems very, very interested in this weapon. "Where did you find it? Can you bring us more? We would gladly make you rich if you do, this is the perfect weapon for training."`
+			choice
+				`	"Sorry, this is one of a kind."`
+			`	"Oh, if you want I can cancel the exchange then, if it is precious to your eyes."`
+			choice
+				`	"<payment> is much more precious, thank you."`
+				`	"Yes I have changed my mind, cancel the exchange."`
+					decline
+			`	"This will be my personal weapon, I will take good care of it worry not!"`
+				accept
+	on accept
+		payment 10000000
+		outfit "Void Rifle" -1
```

Comment:
```suggestion
		outfit "Void Rifle" -1
```

I really like this grouping of alternative completions here. Especially the Void Rifle option.

But with both the Pug and the Coalition being out of the way to obtain weapons from, I feel like there should be some better prompts/hints as to where to get them from. Maybe even hidden behind a timer of say, 120 days-ish, if this hasn't been progressed yet?

Perhaps some mention of tales from over zealous Unfettered crews who went TOO far north, and encountered some strange other alien ships. Something something ~"even those few who succeeded in boarding the strange ships made no progress capturing them; they were repelled by weapons which disoriented and confused the Hai they were used again."

Reply:
I think I will do a hint that triggers after this if you have completed the main plot, mentioning you heard tales of the pugs using these kind of weapons
this mission is not required to do the next one though, it will only come into play later so if the player waits a bit it's no big deal